components/Header/Header.tsx

```diff
@@ -6,11 +6,10 @@ import styled from "styled-components";
 import Menu from "/public/assets/icons/menu.svg";
 
 export function Header() {
-  const { setPanelType, setPanelOpen } = usePanelContext();
+  const { openPanel } = usePanelContext();
```

Comment:
We need more sophisticated control over opening and closing the panel now because we want to allow having the vote panel open up over the history panel when the user clicks on a vote in the history panel. I've extracted this all to a more convenient helper that is also cleaner.